### PR TITLE
get_otp_screen

### PR DESCRIPTION
**Pull request by tanishq-2208**

### CODE REVIEW

File: lib/main.dart
[the remark this answers is not in the document]
Comment (maheshalyana) [reply lib/main.dart:1]: remove any changes in the main.dart

**Review by maheshalyana — CHANGES_REQUESTED** [review]
remove changes from main.dart and we good to go

File: lib/main.dart
[the remark this answers is not in the document]
Comment (tanishq-2208) [reply lib/main.dart:1]: changes done.

**Review by Abhi-0987 — COMMENTED** [review]
image

Font should be changed.. moving to development

**Review by maheshalyana — CHANGES_REQUESTED** [review]
change the font style and move to Review

File: lib/main.dart
Comment (maheshalyana) [inline lib/main.dart:1]: remove changes from main.dart

**Review by maheshalyana — CHANGES_REQUESTED** [review]

File: lib/main.dart
Comment (tanishq-2208) [reply lib/main.dart:1]: Done

**Review by maheshalyana — APPROVED** [review]

**Review by Abhi-0987 — COMMENTED** [review]
image
image

Screen matches with the design and test cases passed.. moving it to acceptance.

**Pull request merged**